packages/core/src/protocol-helpers.ts

```diff
@@ -138,16 +137,13 @@ export default class ProtocolHelpers {
     ): string | undefined {
         // try to find contentType
         if (
-            td &&
-            actionName &&
-            uriScheme &&
-            td.actions &&
-            td.actions[actionName] &&
-            td.actions[actionName].forms &&
+            actionName != null &&
+            uriScheme != null &&
+            td?.actions[actionName]?.forms != null &&
             Array.isArray(td.actions[actionName].forms)
```

Comment:
This change should now actually work (tested it locally):

```suggestion
            td.actions != null &&
            Array.isArray(td.actions[actionName]?.forms)
```